jaeger-core/src/main/java/com/uber/jaeger/Tracer.java

```diff
@@ -305,7 +327,8 @@ boolean isRPCServer() {
               startTimeMicroseconds,
               startTimeNanoTicks,
               computeDurationViaNanoTicks,
-              tags);
+              tags,
+              references);
```

Comment:
one concern I have here is that we're duplicating the chid-of reference by keeping it explicitly as well we recording it implicitly in the parent_id. Strictly speaking in OpenTracing the parent_id is redundant, but we kept it (a) for efficiency since 99% of spans have a simple child-of relationship to the parent that we can represent with a single number instead of a Reference struct, and (b) to main our legacy zipkin-style wire format that includes parent ID (even though with `zipkinSharedRPCSpan=false` we completely ignore the parent_id on the wire). 

On the other hand, in the UI data model we abolished the parent_id and represent everything with References, to keep it normalized. So when we keep this reference here, the code in the backend will create two identical references for the UI model (https://github.com/uber/jaeger/blob/master/model/converter/json/from_domain.go#L87).

We can either fix the backend code to dedupe those, but we could also exclude child-of reference here since it's redundant and only increases the payload.